docs/software/apple/usage.mdx

```diff
@@ -3,6 +3,71 @@ id: usage
 title: Apple Application Usage
 sidebar_label: Usage
 ---
+## Intro
+
+There is a Meshtastic app in the Apple App Store for iPhones, iPads and Macs (on their respective OSs: iOS, iPadOS, and macOS) with slightly different behaviour on each platform. On iOS and iPadOS, V16.2 or greater is required; macOS requires Sonoma.
+
+Let us call the thing with the antenna a "node" for now.
+
+On all platforms the app uses the usual features of the OS, i.e. for notifications (in "Notifications|Meshtastic"), for timestamps (in "General|Language&Region|Date Format"), plus on the Mac there is extra functionality in the menu bar.
+
+The app itself has help in the form of tips for many situations, and we will not duplicate them here.
+
+## First App Start
+
+First, the node should be prepared https://meshtastic.org/docs/getting-started - otherwise the app will look pretty boring - as most of the functionality depends on a connected node.
+
+The app always opens the "Bluetooth" page first. If your node does not show up on the page, make it physically start up now - a simple way is to disconnect power (a.k.a. the USB cable) for a moment. The app should now feel the node and you read a 6-digit number from the node and key it into the app. Then the node should show up as "connected". A green symbol with the node's short name will now show up in the upper right corner - and this indicator will be on each page from now on.
```

Comment:
I'm not sure about recommending a reboot to have the node show up.  That's not something I practice to establish a connection anyways.